.gitignore

```diff
@@ -1,3 +1,4 @@
+/dist/
```

Comment:
All static application files ought be listed

Remember to delete vestigial files